core/Dockerfile.template

```
@@ -11,13 +11,21 @@ COPY package*.json ./
 
 RUN npm ci
 
-FROM balenalib/%%BALENA_ARCH%%-alpine-node:12.19.1-3.12-run-20201211
+FROM balenalib/%%BALENA_ARCH%%-alpine:3.12-run-20211030
```

```suggestion
FROM balenalib/%%BALENA_ARCH%%-alpine-node:12.22.7-bullseye-run-20211207
```
Wouldn't this be preferable?